src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -696,6 +696,14 @@ function InitializeEnvironmentalVariables(frb: FunctionsRuntimeBundle): void {
   process.env.GCLOUD_PROJECT = frb.projectId;
   process.env.FUNCTIONS_EMULATOR = "true";
 
+  const nodeVersion = parseVersionString(process.versions.node);
+  if (nodeVersion.major >= 10) {
+    const triggerId = frb.triggerId || "";
+    process.env.FUNCTION_TARGET = triggerId.replace(/-/g, ".");
```

Comment:
Could you add a comment with a link to the documentation that talks about these env variables so that future code authors understand why this is here?